vickev/howdypix
# Fetch the root sources in the first page

## Description

Fetches the root sources from a GraphQL query to display them.

![image](https://user-images.githubusercontent.com/219680/69495035-5ad1cd80-0e90-11ea-91e9-179e98f8c8e2.png)






## How to test?

- Check-out this branch.
- Run the project => you should see the root sources

## Checklist
